go-fuzz-build/main.go

```diff
@@ -24,6 +24,7 @@ import (
 )
 
 var (
+	flagTag  = flag.String("tags", "", "a list of build tags to consider satisfied during the build.")
```

Comment:
Now that I'm looking at it I'm not sure I want to pull in a new package just for a flag. I'll let @dvyukov decide on whether he'd like this or not.